contracts/ICS26Router.sol

```diff
@@ -305,6 +304,20 @@ contract ICS26Router is
         emit TimeoutPacket(msg_.packet.sourceClient, msg_.packet.sequence, msg_.packet);
     }
 
+    /// @inheritdoc IICS26Router
+    function grantPortCustomizerRole(address account) external onlyAdmin {
+        _grantRole(PORT_CUSTOMIZER_ROLE, account);
+    }
+
+    /// @inheritdoc IICS26Router
+    function revokePortCustomizerRole(address account) external onlyAdmin {
+        _revokeRole(PORT_CUSTOMIZER_ROLE, account);
+    }
+
+    /// @inheritdoc ICS02ClientUpgradeable
+    function _authorizeSetLightClientMigratorRole(string calldata, address) internal view override onlyAdmin { }
```

Comment:
This tells me we have some slightly off abstractions in our code.

Clearly ICS02Client is supposed to have an onlyAdmin concept, but it currently lives in the IBCUUPSUpgradable. I think the need to do this shows that we should reconsider where that lives. Perhaps it's an abstract "admin" contract with the onlyAdmin. Not 100% sure how it should look and be structured, but I think onlyAdmin should be available to ICS02Client so we don't have to do this kind of roundabout hacks.

Reply:
I don't think this is a roundabout hack. This is what I intended with the design, UUPS upgradeability works the same way. ICS02 need not know anything about the admins

Reply:
I see. I can see now that this is the same pattern. This is where inheritance is starting to lose its appeal. This will work, and it's probably the right way to do it, but all this back and forth required for these things are really bad for maintainability. But that is probably more of a solidity issue than anything else. Anyway. This is fine for now, then 👍